src/bp/bpfs.ts

```diff
@@ -107,13 +108,26 @@ class BPFS {
       const archive = await createArchiveFromFolder(this.sourceDir, pushedArchiveIgnoredFiles)
       const axiosClient = this._getPushAxiosClient(archive.length)
 
+      if (useForce) {
+        console.info(chalk.blue(`Force pushing local changes to ${this.serverUrl}...`))
```

Comment:
would be nice to have feedback on progress here as well

Reply:
I tried, but onUploadProgress is only working when you use axios on the browser. To use on node, we need to use streams and check the progress instead. Keeping this as a future possible update

Reply:
👍